### PR TITLE
support train_size of holdout resampling_strategy

### PR DESCRIPTION
**Pull request by caoyi0905**
same to the #261 ,but in development branch.

### CODE REVIEW

File: autosklearn/evaluation/__init__.py [hunk 1]
```diff
@@ -234,21 +234,24 @@ def run(self, config, instance=None,
 
     def get_splitter(self, D):
         y = D.data['Y_train'].ravel()
-
+        train_size = 0.67
+        if not self.resampling_strategy_args and self.resampling_strategy_args.get('train_size'):
```
Comment (mfeurer) [inline autosklearn/evaluation/__init__.py]: Shouldn't this rather be `if self.resampling_strategy_args and self.resampling_strategy_args.get('train_size'):`? 

And how about simplifying it?

```
if self.resampling_strategy_args:
    train_size = self.resampling_strategy_args.get('train_size', train_size)
```

Comment (caoyi0905) [inline autosklearn/evaluation/__init__.py]: Thanks for check my code ,I learned a lot from you pythonic code,thanks again.

**Comment by mfeurer** [comment]
Thank you very much.

**Comment by caoyi0905** [comment]
@mfeurer By the way, I found some [code](https://github.com/automl/auto-sklearn/blob/development/autosklearn/metalearning/metalearning/metrics/result_correlation.py#L28) that seems unnecessary.Am I right?

**Comment by mfeurer** [comment]
Correct, that code isn't used. But better don't spend time on changing parts in the meta-learning module if you don't want to do anything in there. Maybe you want to give me a brief overview of what you want to achieve.

**Comment by caoyi0905** [comment]
@mfeurer  I want to save metafeatures which I have trained by autosklearn, and let automl use those in meta-learning module.

**Comment by mfeurer** [comment]
I'm not sure if I understand what you did. Did you calculate meta-features based on the scripts in auto-sklearn on new datasets and want to use them now?

**Pull request merged**